.webpack.config.js

```diff
@@ -172,6 +172,11 @@ var filesToCopy = [
         from: '{fonts/*,tabler-icons.scss}',
         to: scssOutputPath,
     },
+    // {
```

Comment:
Should not be commented or not be added :)

Reply:
yeah yeah, I commited to discuss it with Cédric